main-app/tests/test.test.js

```diff
@@ -0,0 +1,51 @@
+(function () {
+    'use strict';
+
+    describe('GameServerProxy tests', function(){
+        var gameServerProxy, $httpBackend, requestHandler;
+
+        beforeEach(function(){
+            module('Tombola.Games.NoughtsAndCrosses.API');
+
+            inject(function($injector){
+                gameServerProxy = $injector.get('GameServerProxy');
+                // Set up the mock http service responses
+                $httpBackend = $injector.get('$httpBackend');
+                // backend definition common for all tests
+                requestHandler = $httpBackend.when('POST', 'http://eutaveg-01.tombola.emea:35000/api/v1.0/makemove')
```

Comment:
No  - this is per-"IT" test. You are overrifing ewith $httpBackend.expectPOST in the it statements.
You should be able to remove this.